docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description**: The level of the logs to be printed. This configuration item is used to control the output of logs initiated with VLOG in codes.
+
+#### log_buffer_level
+
+- **Default**: Empty string
+- **Unit**: N/A
+- **Description**: The strategy for flushing logs. The default value indicates that logs are buffered in memory. Valid values are -1 and 0. -1 indicates that logs are not buffered in memory.
+
+#### num_threads_per_core
+
+- **Default**: 3
+- **Unit**: N/A
+- **Description**: The number of threads started on each CPU core.
+
+#### compress_rowbatches
+
+- **Default**: TRUE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to compress the row batches in RPCs between BEs. TRUE indicates compressing the row batches, and FALSE indicates not compressing them.
+
+#### serialize_batch
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to serialize the row batches in RPCs between BEs. TRUE indicates serializing the row batches, and FALSE indicates not serializing them.
+
+#### storage_root_path
+
+- **Default**: `${STARROCKS_HOME}/storage`
+- **Unit**: N/A
+- **Description**: The directory and medium of the storage volume.
+  - Multiple volumes are separated by semicolons (`;`).
+  - If the storage medium is SSD, add `medium:ssd` at the end of the directory.
+  - If the storage medium is HDD, add ,medium:hdd at the end of the directory.
+
+#### max_length_for_bitmap_function
+
+- **Default**: 1000000
+- **Unit**: Byte
+- **Description**: The maximum length of input values for bitmap functions.
+
+#### max_length_for_to_base64
+
+- **Default**: 200000
+- **Unit**: Byte
+- **Description**: The maximum length of input values for the to_base64() function.
+
+#### max_tablet_num_per_shard
+
+- **Default**: 1024
+- **Unit**: N/A
+- **Description**: The maximum number of tablets in each shard. This configuration item is used to restrict the number of tablet child directories under each storage directory.
+
+#### max_garbage_sweep_interval
+
+- **Default**: 3600
+- **Unit**: Second
+- **Description**: The maximum time interval for garbage collection on storage volumes.
+
+#### min_garbage_sweep_interval
+
+- **Default**: 180
+- **Unit**: Second
+- **Description**: The minimum time interval for garbage collection on storage volumes.
+
+#### file_descriptor_cache_capacity
+
+- **Default**: 16384
+- **Unit**: N/A
+- **Description**: The number of file descriptors that can be cached.
+
+#### min_file_descriptor_number
+
+- **Default**: 60000
+- **Unit**: N/A
+- **Description**: The minimum number of file descriptors in the BE process.
+
+#### index_stream_cache_capacity
+
+- **Default**: 10737418240
+- **Unit**: Byte
+- **Description**: The cache capacity for the statistical information of BloomFilter, Min, and Max.
+
+#### disable_storage_page_cache
+
+- **Default**: FALSE
+- **Unit**: N/A
+- **Description**: A boolean value to control whether to disable PageCache.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/be45e71d-5b18-4e54-8898-d55f653ab173)
Delete "A boolean value to control", so the sentence starts with "Whether".